### DISCUSSION TITLE
Unreadable display of checkboxes in the dark theme

### DISCUSSION THREAD

**Discussion by chrisonoo** [community/community#61732] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hey, 
I've noticed an error in the formatting of checkbox in GitHub's dark theme. Below I will show how the lists look in different browsers. It seems that only Firefox nicely renders this input.

## Firefox
![firefox](https://github.com/community/community/assets/125088287/0d890261-020b-460b-969d-8f1f0f2aece9)

## Chrome
![chrome](https://github.com/community/community/assets/125088287/7e6bd8d0-9694-461b-9748-4f390cab9e49)

## Edge
![edge](https://github.com/community/community/assets/125088287/fe5fa4cb-9fb7-4c3f-9b18-8b52b4b8da58)

**Comment by NicolaIsotta** [comment]
Still an issue, this is Edge 135:
![image](https://github.com/user-attachments/assets/321ecd87-d768-4f0e-a1e6-78ad9cc8c9af)

**Comment by GHXX** [comment]
Seems to also still be an issue on desktop in Opera 119. Likeeee, its _incredibly_ difficult for me to tell the difference of disabled checkboxes being checked or not, but see for yourselves:
![image](https://github.com/user-attachments/assets/50c1cf74-7ea6-4db5-b313-b834ca48e251)

**Comment by saoud30** [comment]
You're absolutely right — checkbox rendering in Markdown (especially in dark mode) can look inconsistent across browsers.

- What’s happening:
GitHub renders checkboxes using HTML inputs styled by the browser's default rendering engine. Since each browser (like Chrome, Firefox, Safari) has its own way of displaying form elements, things like checkboxes often look slightly different — and GitHub doesn't fully override those native styles, especially in dark mode.

In Firefox, the checkbox style blends well with GitHub’s dark theme, but in Chromium-based browsers (like Chrome or Edge), they often appear with mismatched spacing, outlines, or size.

- Possible Workarounds:
Use emojis instead of checkboxes if you just want a clean visual list:

arduino
Copy
Edit
🔲 Task 1  
☑️ Task 2
Avoid checkbox markdown (- [ ]) if pixel-perfect alignment matters, and use plain lists for consistency.

For developers/stylers: You could create a browser extension or user style (via Stylus) that overrides GitHub’s checkbox styles in dark mode.

**Reply by GHXX** [reply]
Changing the text of the markdown todo list isnt really an option often in my opinion, as, usually when you are reading the list, you are not the one that made it (because then you would already know the content and be editing it in markdown-source), and thus, normally, cant edit it.

A quick ublock rule that adds some css does work though as a workaround. Hopefully github fixes it for good though at some point.
